.github/ISSUE_TEMPLATE/bug_report_v2.yaml

```diff
@@ -32,30 +32,30 @@ body:
     attributes:
       label: Operating System
       description: Describe your operating system
```

Comment:
Just occurred to me, maybe we can add: "If your development operating system is different from the user's operating system, describe both and indicate which one the bug is occurring in."

Reply:
I changed this to `Describe the operating system(s) where you are experiencing the issue.`